SIL.Media/SIL.Media-Designer.csproj

```diff
@@ -85,12 +85,12 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="AudioIrrKlangSession.cs" />
+    <Compile Include="IAudioPlayer.cs" />
```

Comment:
It looks like some of the files in SIL.Media were moved around, but the Designer project was not updated.